tdrs-backend/tdpservice/reports/test/conftest.py

```diff
@@ -1,19 +1,17 @@
 """Report file fixtures."""
 
-import uuid
 import pytest
 
 @pytest.fixture
-def report_data():
+def report_data(user):
     """Return report creation data."""
     return {
-        "id": uuid.uuid4(),
         "original_filename": "report.txt",
         "slug": "report-txt-slug",
         "extension": "txt",
         "section": "Active Case Data",
+        "user": str(user.id),
         "quarter": "Q1",
         "year": "2020",
```

Comment:
We changed the "year" field to an integer ... but I guess if you pass in a string the instance is still valid?

Reply:
I'll clear that up here, the tests do pass with the string, but it would make sense to match the change in migration

Reply:
Looks like it's still a string in `conftest.py`?

Reply:
I just updated it @alexsoble

Reply:
Hmmm, maybe you updated it on the Raft repo but not here? I'm not seeing the change here